htdocs/web_portal/controllers/service_group/add_service_group.php

```diff
@@ -67,16 +69,30 @@ function submit($user) {
     }
 }
 
-function draw($user) {
+function draw($user)
+{
     try {
-        // If the user is registered they're allowed to add a service group
```

Comment:
as this comment is getting removed, maybe some new comments need to be added

Reply:
I think variable names are giving the enough information the developer needs. So, I don't think a new comment is needed here. Does that sounds good?